frictionless/errors/row.py

```diff
@@ -66,11 +66,72 @@ class PrimaryKeyError(RowError):
     template = 'Row at position "{rowNumber}" violates the primary key: {note}'
 
 
-class ForeignKeyError(RowError):
+@attrs.define(kw_only=True)
+class ForeignKeyError(TableError):
     type = "foreign-key"
     title = "ForeignKey Error"
     description = "Values in the foreign key fields should exist in the reference table"
     template = 'Row at position "{rowNumber}" violates the foreign key: {note}'
+    tags = ["#table", "#row"]
+
+    cells: List[str]
+    """NOTE: add docs"""
+
+    row_number: int
+    """NOTE: add docs"""
+
+    target_keys: List[str]
+    """NOTE: add docs"""
+
+    source_keys: List[int]
+    """NOTE: add docs"""
+
+    source_name: List[str]
+    """NOTE: add docs"""
+
+    missing_values: List[int]
```

Comment:
Sorry not sure what `missing_values` means in this context

Reply:
I added a description in the docstrings:
"Cell values not found in the lookup table."